ingenialink/canopen/dictionary.py

```diff
@@ -1,86 +1,9 @@
-import xml.etree.ElementTree as ET
-from ..dictionary import Dictionary
-from ..constants import SINGLE_AXIS_MINIMUM_SUBNODES
-from .register import CanopenRegister, REG_ACCESS, REG_DTYPE
-from ingenialink.utils._utils import *
-from .._ingenialink import lib
+from ingenialink.dictionary import Dictionary
+from ingenialink.constants import SINGLE_AXIS_MINIMUM_SUBNODES
+from ingenialink.canopen.register import CanopenRegister
+import ingenialogger
```

Comment:
This is a third-party library. It should be on top of the local library imports and separated with a blank line.